src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php

```diff
@@ -167,7 +167,7 @@ protected function getMappingDriverBundleConfigDefaults(array $bundleConfig, \Re
     /**
      * Register all the collected mapping information with the object manager by registering the appropriate mapping drivers.
      *
-     * @param ContainerBuilder $container     A ContainerBuilder instance
+     * @param ContainerBuilder $container A ContainerBuilder instance
```

Comment:
this one can be completely removed in fact :)

Reply:
Let's automate it :) 

Remove all messages in format:

*A <type> instance*

What do you think?